setup.py

```diff
@@ -1,17 +1,33 @@
+from subprocess import check_output
+
 from setuptools import setup, find_packages
 
+def get_version():
+    # https://github.com/uc-cdis/dictionaryutils/pull/37#discussion_r257898408
+    try:
+        tag = check_output(
+            ["git", "describe", "--tags", "--abbrev=0", "--match=[0-9]*"]
+        )
+        return tag.decode("utf-8").strip("\n")
+    except Exception:
+        raise RuntimeError(
+            "The version number cannot be extracted from git tag in this source "
+            "distribution; please either download the source from PyPI, or check out "
+            "from GitHub and make sure that the git CLI is available."
+        )
+
+
 setup(
     name='gdcdictionary',
-    version='0.0.0',
+    version=get_version(),
+    description="Gen3 generic data dictionary",
+    license="Apache",
     packages=find_packages(),
     install_requires=[
-        'PyYAML==3.11',
-        'jsonschema==2.5.1',
+        'PyYAML>=3.11',
+        'jsonschema>=2.5.1',
```

Comment:
But pls keep this one as it is because I'm not sure we want a full upgrade to 3.x or stay with the 2.x series.